x-pack/plugins/observability/public/embeddable/slo/alerts/slo_alerts_embeddable.tsx

```diff
@@ -0,0 +1,147 @@
+
+import {
+  type CoreStart,
+  IUiSettingsClient,
+  ApplicationStart,
+  NotificationsStart,
+} from '@kbn/core/public';
+import { TriggersAndActionsUIPublicPluginStart } from '@kbn/triggers-actions-ui-plugin/public';
+import type { DataPublicPluginStart } from '@kbn/data-plugin/public';
+import { Router } from '@kbn/shared-ux-router';
+import { SettingsStart } from '@kbn/core-ui-settings-browser';
+import { SecurityPluginStart } from '@kbn/security-plugin/public';
+import type { ChartsPluginStart } from '@kbn/charts-plugin/public';
+import type { UiActionsStart } from '@kbn/ui-actions-plugin/public';
+import { ServerlessPluginStart } from '@kbn/serverless/public';
+
+import { SloAlertsWrapper } from './slo_alerts_wrapper';
+import type { SloAlertsEmbeddableInput } from './types';
+export const SLO_ALERTS_EMBEDDABLE = 'SLO_ALERTS_EMBEDDABLE';
+
+const history = createBrowserHistory();
+
+export interface SloEmbeddableDeps {
+  uiSettings: IUiSettingsClient;
+  http: CoreStart['http'];
+  i18n: CoreStart['i18n'];
+  application: ApplicationStart;
+  triggersActionsUi: TriggersAndActionsUIPublicPluginStart;
+  data: DataPublicPluginStart;
+  notifications: NotificationsStart;
+  cases: CasesUiStart;
+  settings: SettingsStart;
+  security: SecurityPluginStart;
+  charts: ChartsPluginStart;
+  uiActions: UiActionsStart;
+  serverless?: ServerlessPluginStart;
+}
+
+export class SLOAlertsEmbeddable extends AbstractEmbeddable<
+  SloAlertsEmbeddableInput,
+  EmbeddableOutput
+> {
+  public readonly type = SLO_ALERTS_EMBEDDABLE;
+  private subscription: Subscription;
+  private node?: HTMLElement;
+
+  constructor(
+    private readonly deps: SloEmbeddableDeps,
+    initialInput: SloAlertsEmbeddableInput,
+    parent?: IContainer
+  ) {
+    super(initialInput, {}, parent);
+    this.deps = deps;
+
+    this.subscription = new Subscription();
+    this.subscription.add(this.getInput$().subscribe(() => this.reload()));
+  }
+
+  public reportsEmbeddableLoad() {
+    return true;
+  }
+
+  public onRenderComplete() {
+    this.renderComplete.dispatchComplete();
+  }
+
+  setTitle(title: string) {
+    this.updateInput({ title });
+  }
+
+  public render(node: HTMLElement) {
+    super.render(node);
+    this.node = node;
+    // required for the export feature to work
+    this.node.setAttribute('data-shared-item', '');
+    this.setTitle(
+      this.input.title ||
+        i18n.translate('xpack.observability.sloAlertsEmbeddable.displayTitle', {
+          defaultMessage: 'SLO Alerts',
+        })
+    );
+    const queryClient = new QueryClient();
+
+    const I18nContext = this.deps.i18n.Context;
+    const { slos, timeRange = { from: 'now-15m/m', to: 'now' } } = this.getInput();
+    const deps = this.deps;
+    ReactDOM.render(
+      <I18nContext>
+        <KibanaContextProvider
+          services={{
+            ...this.deps,
+            storage: new Storage(localStorage),
+            isServerless: !!deps.serverless,
```

Comment:
I think I need to pass `kibanaVersion` as well similar to how it is done [here](https://github.com/elastic/kibana/blob/main/x-pack/plugins/observability/public/application/index.tsx#L101). I don't know the details and where it might be needed for (possibly rule_registry, alerting, saved_objects), but it looks important to me. I didn't know how to get this information, since I am on the embeddable context and not the [observability context ](https://github.com/elastic/kibana/blob/main/x-pack/plugins/observability/public/plugin.ts#L222). I need help on this.